x/inflation/keeper/hooks_test.go

```diff
@@ -146,10 +156,29 @@ func (suite *KeeperTestSuite) TestPeriodChangesAfterEpochEnd() {
 			"[Period 1] with skipped epochs - period changes once enough epochs have passed",
 			1,
 			2*currentEpochPeriod + 11,
+			epochstypes.DayEpochID,
 			10,
 			true,
 			true,
 		},
+		{
```

Comment:
Can you move the skipped epoch tests up, so that the test order follows the order of the code?